notebooks/optimal-stopping-point/seldon-deployment-config.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: osp-seldon
+spec:
+  name: osp
+  annotations:
+    seldon.io/rest-timeout: "1000000"
+    seldon.io/grpc-timeout: "1000000"
+  predictors:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: quay.io/kachau/optimal-stopping-point:latest
```

Comment:
After speaking to @harshad16, we saw that there are issues in moving this to aicoe for the moment because [this](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.aicoe-ci.yaml) is already set. We are planning to fix this in a separate PR since there are issues building this now.